### DISCUSSION TITLE
Thoughts on creating a new project and the Create Folder button

### DISCUSSION THREAD

**Discussion by kpdyer16** [godotengine/godot-proposals#8846] [Editor]
Let me preface this by saying, I'm a new user to Godot, and this is my first time starting a discussion here. Please redirect me if this doesn't need to be a new discussion. My main issue is addressed in #6070, but not mentioned in there is the default behavior of _not_ creating a new folder for a new project. Again, let me know if my discussion should go in there or somewhere else.

Is there an intentional reason why it doesn't automatically create a new folder for your project? As I go through Godot's getting started tutorials and testing out behaviors, I find myself creating lots of new projects, and this has sort of bothered me. Godot's create new project dialog even warns you if the selected path is not empty, so it seems like the recommended decision is to create a new folder for your project. But you have to have either already created it in your file browser/terminal,etc. or you have to click the Create Folder button each time you create a new project. 

To me, a better default is to automatically create a new folder and to offer the choice to create the project in the current folder.

**Comment by Calinou** [comment]
> To me, a better default is to automatically create a new folder and to offer the choice to create the project in the current folder.

This is being addressed in https://github.com/godotengine/godot/pull/56420.

**Reply by kpdyer16** [reply]
Thanks for the redirect!

**Comment by tetrapod00** [comment]
The linked PR was merged, closing as resolved
